code/modules/vore/eating/belly_obj_vr.dm

```diff
@@ -38,7 +38,7 @@
 	//Actual full digest modes
 	var/tmp/static/list/digest_modes = list(DM_HOLD,DM_DIGEST,DM_ABSORB,DM_DRAIN,DM_UNABSORB,DM_HEAL,DM_SHRINK,DM_GROW,DM_SIZE_STEAL)
 	//Digest mode addon flags
-	var/tmp/static/list/mode_flag_list = list("Numbing" = DM_FLAG_NUMBING, "Stripping" = DM_FLAG_STRIPPING, "Leave Remains" = DM_FLAG_LEAVEREMAINS)
+	var/tmp/static/list/mode_flag_list = list("Numbing" = DM_FLAG_NUMBING, "Stripping" = DM_FLAG_STRIPPING, "Leave Remains" = DM_FLAG_LEAVEREMAINS, "Thick Belly" = DM_FLAG_THICKBELLY)
```

Comment:
I think, while thick belly is a good description, its very un-intuitive in terms of "what does that do?" aspect of the name. Muffling or something along those lines would be much better in terms of explanation of "what this flag actually does"